com.kassindornelles.lolforlinuxinstaller.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: com.kassindornelles.lolforlinuxinstaller
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-22.08
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+finish-args:
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+command: com.kassindornelles.lolforlinuxinstaller
+modules:
+    - name: PyQtApp
+    - python3-modules.json
+    - name: lol-for-linux
+      buildsystem: simple
+      cleanup:
+        - '.git'
+        - '.github'
+        - '.gitignore'
+        - 'license.md'
+        - './src'
+        - 'wine_build.json'
+        - 'README.md'
+
+      build-commands:
+        #install the launcher to the correctly location
+        - install -D ./src/com.kassindornelles.lolforlinuxinstaller.py /app/bin/com.kassindornelles.lolforlinuxinstaller
+
+        # install the icon and .desktop to the correctly location
+        - install -D ./src/com.kassindornelles.lolforlinuxinstaller.desktop /app/share/applications/com.kassindornelles.lolforlinuxinstaller.desktop
+        - install -D ./src/lolforlinuxinstaller.svg /app/share/icons/hicolor/scalable/apps/com.kassindornelles.lolforlinuxinstaller.svg
+
+        #edit the .desktop to point to the new icon name and launcher
+        - desktop-file-edit --set-key="Icon" --set-value="com.kassindornelles.lolforlinuxinstaller" /app/share/applications/com.kassindornelles.lolforlinuxinstaller.desktop
+        - desktop-file-edit --set-key="Exec" --set-value="com.kassindornelles.lolforlinuxinstaller %F" /app/share/applications/com.kassindornelles.lolforlinuxinstaller.desktop
+
+        #install the libraries to the correctly location
+        - install -D ./src/launch-script.py /app/usr/share/lol-for-linux-installer/launch-script.py
```

Comment:
`/app/usr/share` is INCORRECT. It should be `/app/share`

Reply:
it is because his proggram was looking for /usr/* files, so it was easier to make that way, it's necessary to change?

Reply:
you shouldn't even hardcode `/usr` anywhere.